tests/h/search/query_test.py

```diff
@@ -944,7 +944,7 @@ def test_matches_replies_of_replies_to_an_annotation(self, Annotation, search):
 
 class TestTagsAggregation:
     def test_it_returns_annotation_counts_by_tag(self, Annotation, search):
-        for i in range(2):
+        for _ in range(2):
             Annotation(tags=["tag_a"])
```

Comment:
This looks a bit odd. Is this a factory fixture? If so `create_batch` or something might be better.

(Can you tell I'm only reviewing this from the diff?)